README.md

```diff
@@ -204,6 +204,8 @@ grunt custom --name="myPackage"
 #### External Libraries
 
 * fastXmlPull
+* fonts
```

Comment:
Which fonts? What library does this refer to? We should really add links to these list elements and also revisit the text at the base which says "All of these libraries are already bundled within the Infusion image."

Reply:
The fonts are referring to [open-sans](https://github.com/fluid-project/infusion/blob/master/package.json#L62). 

When you say add links, what would you like to point at? The same place we point links to for the libraries in the release notes? or somewhere else?

For "All of these libraries are already bundled within the Infusion image." Maybe we should just remove that line. I'm not sure it's really adding anything anyways.

Reply:
Well we should remove the line since it is false :)
We can just rewrite the line "fonts" to read opensans-webkit since then readers will have some guess what this refers to

Reply:
I guess it was fonts because we were expecting it to possibly have more, but i can change that up.

Reply:
I found at least one more font that should be pulled in by NPM, Roboto, and will also update them to use woff instead of ttf ( see: https://issues.fluidproject.org/browse/FLUID-6354 ).

Reply:
@amb26 I've made a bunch of improvements to the sourcing of fonts, including icon fonts. I've also switched, where possible, fonts to woff as these are intended to be more performant on the web. Please not that my update to the overview panel icons is based off of changes I've made in https://github.com/fluid-project/infusion-icons/pull/7